.pylintrc

```diff
@@ -187,6 +187,7 @@ function-naming-style=snake_case
 # Good variable names which should always be accepted, separated by a comma.
 good-names=FlightPhases,
            WindroseAxes,
+           barometric_height_ISA,
```

Comment:
Maybe there are more variables that could be added to this list to reduce the numbers of pylint errors?

Reply:
Solved here: [91229c0](https://github.com/RocketPy-Team/RocketPy/pull/550/commits/91229c03dc6970ade080dbd4fa008393fb8753a6)